<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>691</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>461</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_hotel">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_name"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_sterne">
       <property name="text">
        <string>Sterne</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_sterne"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_strasse">
       <property name="text">
        <string>Strasse</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_anzahl_zimmer">
       <property name="text">
        <string>Anzahl RÃ¤ume</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QComboBox" name="comboBox_anzahl_zimmer"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_strasse"/>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_PLZ">
       <property name="text">
        <string>PLZ</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lineEdit_plz"/>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Ort</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="lineEdit_ort"/>
     </item>
     <item row="12" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>50</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_speichern">
         <property name="text">
          <string>Speichern</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_abbrechen">
         <property name="text">
          <string>Abbrechen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_clear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_header">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Neues Hotel eingeben</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
